packages/grafbase-sdk/src/resolver/context.ts

```diff
@@ -0,0 +1,68 @@
+import { KVListResult, KVMetadata } from './context/kv'
+import { Classification } from './context/ai'
+
+/**
+ * The type of the `context` argument in a Grafbase edge resolver.
+ *
+ * Reference: https://grafbase.com/docs/edge-gateway/resolvers#context
+ *
+ * @example
+ *
+ * import { Context, Info } from '@grafbase/sdk'
+ *
+ * export default async function(_parent, _args, context: Context) {
+ *   // ...
+ * }
+ */
+export type ResolverContext = {
+  /** Context about the HTTP request being handled. */
+  request: {
+    headers: Record<string, any>,
+  }
+  /**
+   * Grafbase KV
+   *
+   * If you want to use this, please make sure to [enable KV in your Grafbase configuration](https://grafbase.com/docs/edge-gateway/resolvers#enable-kv).
+   *
+   * See the reference documentation: https://grafbase.com/docs/edge-gateway/resolvers#kv
+   */
+  kv: {
+    /** Retrieve the value and metadata for a key. See [the docs](https://grafbase.com/docs/edge-gateway/resolvers#get) for examples. */
+    get: (key: string) => Promise<{ metadata: KVMetadata, value: any }>
+    /** Create a new key-value pair or update an existing one. See [the docs](https://grafbase.com/docs/edge-gateway/resolvers#set) for examples. */
+    set: (key: string, value: any, metadata?: KVMetadata) => Promise<void>
```

Comment:
The `metadata` is nested inside this object, this arg also has `ttl` property.

Reply:
Indeed, found the definition for these and fixed that. Thanks!